packages/react-query/src/shared/proxy/useQueriesProxy.ts

```diff
@@ -65,15 +65,14 @@ export function createUseQueriesProxy<TRouter extends AnyRouter>(
   client: TRPCUntypedClient<TRouter>,
 ) {
   return createRecursiveProxy((opts) => {
-    const path = opts.path.join('.');
+    const arrayPath = opts.path;
+    const dotPath = arrayPath.join('.');
     const [input, ...rest] = opts.args;
 
-    const queryKey = getQueryKey(path, input);
-
     const options: QueryOptions = {
-      queryKey,
+      queryKey: getQueryKey(arrayPath, input, 'query'),
       queryFn: () => {
-        return client.query(path, input);
+        return client.query(dotPath, input);
```

Comment:
@sachinraja any reason we aren't forwarding options to the client here?

in the other hooks we do 

```ts
client.query(...getClientArgs(...));
```

Reply:
my 5 second analysis is that I think that's just a miss in the implementation